sites/en/job-board/update_job_listings.step

```diff
@@ -156,7 +156,10 @@ message <<-MARKDOWN
 
   ### Add a Link
 
-  Our users probably aren't going to know they can hit `/jobs/:id/edit` to visit the edit form, so let's add a link to it on the jobs index so we end up with this (we're just adding the line with the `<h5>` header in it ... don't copy and paste the whole thing!):
+  Our users probably aren't going to know they can hit `/jobs/:id/edit` to visit the edit form, so let's add a link so it's easy to find! 
+  
+In `app/views/jobs/index.html.erb`, just add this line with the `<h5>` header in it ... don't copy and paste the whole thing!
```

Comment:
Hi @lorainekv! This line needs to be indented the same as the rest of the block of markdown, or else the markdown stops getting parsed correctly:

![screen shot 2016-07-01 at 7 45 31 pm](https://cloud.githubusercontent.com/assets/978428/16537840/67d1a878-3fc4-11e6-8d11-86918b7c2b13.png)